transportable-udfs-trino-plugin/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'java'
+apply plugin: 'distribution'
+
+java {
+  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+
+dependencies {
+  implementation project(':transportable-udfs-api')
+  implementation project(":transportable-udfs-trino")
+  implementation (group:'io.trino', name: 'trino-main', version: project.ext.'trino-version')
+  compileOnly(group:'io.trino', name: 'trino-spi', version: project.ext.'trino-version')
+}
+
+// packaging as a shaded jar following the guideline from Trino plugin
```

Comment:
Is there a public link for guideline from Trino plugin? If so, could you add it here? Would be great to pin the specific part of shading.

Reply:
I am afraid not. I follow the code example provided by Trino team.